main.yml

```diff
@@ -4,6 +4,7 @@
 # Main playbook for launching guest OS validation testing on vSphere
 - name: main
   hosts: localhost
+  gather_facts: "{{ not no_need_log | default(true) }}"
```

Comment:
I think here can directly set `gather_facts: false` as we don't use local ansible facts in our tests.

Reply:
Sure.